packages/compose/src/hooks/use-media-query/test/index.js

```diff
@@ -40,9 +40,11 @@ describe( 'useMediaQuery', () => {
 			matches: true,
 		} );
 
-		const root = create( <TestComponent query="(min-width: 782px)" /> );
+		const { container } = render(
+			<TestComponent query="(min-width: 782px)" />
+		);
 
-		expect( root.toJSON() ).toBe( 'useMediaQuery: true' );
+		expect( container ).toHaveTextContent( 'useMediaQuery: true' );
 	} );
 
 	it( 'should return true when query matches', async () => {
```

Comment:
I think we should be able to remove the above test in favor of this one - they are almost identical, this one being a little bit more detailed.